purchasing_test/unit/opportunities/test_opportunities.py

```diff
@@ -71,6 +71,35 @@ def setUp(self):
         # import our test categories
         import_nigp(current_app.config.get('PROJECT_ROOT') + '/purchasing_test/mock/nigp.csv')
 
+    def test_templates(self):
```

Comment:
Did you also want to add tests for other places this might be referenced (conductor, opportunities_admin?)